sdk/tests/conformance2/rendering/vertex-id.html

```diff
@@ -14,7 +14,7 @@
 <!-- Shaders for testing instanced draws -->
 <script id="vs" type="text/plain">
 #version 300 es
-flat out int vVertexID;
+flat out highp int vVertexID;
```

Comment:
Vertex shaders default to highp int.

Reply:
Then the problem must have been that the fragment shader didn't declare highp?